llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2238,6 +2242,10 @@ template <> Value *SPIRVToLLVM::transValueWithOpcode<OpStore>(SPIRVValue *const
   SPIRVStore *const spvStore = static_cast<SPIRVStore *>(spvValue);
 
   bool isVolatile = spvStore->SPIRVMemoryAccess::isVolatile(false);
+  const Vkgc::ExtendedRobustness &extendedRobustness =
+      static_cast<Llpc::Context *>(m_context)->getPipelineContext()->getPipelineOptions()->extendedRobustness;
+  if (extendedRobustness.nullDescriptor || extendedRobustness.robustBufferAccess)
+    isVolatile = spvStore->getDst()->isVariable();
```

Comment:
`isVolatile = spvStore->getDst()->isVariable();`

Is it expected that "isVariable()" is queried here, not "isVolatile()"?

Reply:
It should be isVolatile(). I made a mistake. I corrected it in PR #763. Thank you for pointing it out.